mongo_wrapper_meta.c

```diff
@@ -281,6 +281,13 @@ BsonIterDouble(BSON_ITERATOR *it)
 }
 
 
+bool
+BsonIterDecimal128(BSON_ITERATOR *it, bson_decimal128_t *dec)
+{
+  return bson_iter_decimal128(it, dec);
```

Comment:
Can you please format this like the rest of the file?

Reply:
I think the issue is that the PR creator has used spaces (indentation level of 2) whereas the codebase uses tabs. Maybe the PR creator has their editor set up to display tabs at the width of two spaces

Reply:
sorry, i'll have to leave the pleasure to someone else